inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp

```diff
@@ -4,8 +4,12 @@
 
 #include "vpu/utils/runtime_graph.hpp"
 
+#include "ie_ngraph_utils.hpp"
```

Comment:
Not really important but `ie_ngraph_utils.hpp` includes `string`, so maybe line 16 is redundant now.